contracts/registry/src/lib.rs

```diff
@@ -2650,4 +2650,37 @@ mod tests {
         assert_eq!(ctr.sbt_supply(issuer2()), 20);
         assert_eq!(ctr.sbt_supply(issuer3()), 20);
     }
+
+    #[test]
+    fn is_human_call() {
+        let (mut ctx, mut ctr) = setup(&fractal_mainnet(), 150 * MINT_DEPOSIT);
+        ctx.current_account_id = AccountId::new_unchecked("registry.i-am-human.near".to_string());
```

Comment:
why do we change the current account here? I think it's not needed, the predecessor is set to fractal in the setup.